modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -0,0 +1,1200 @@
+				// 	'{{WRAPPER}} .elementor-tab-title' => 'border-color: {{VALUE}};',
+				// ],
+			]
+		);
+
+		$this->add_control(
+			'cta_links_corners',
+			[
+				'label'   => esc_html__( 'Corners', 'elementor' ),
+				'type'    => Controls_Manager::SELECT,
+				'default' => 'round',
+				'options' => [
+					'round'   => esc_html__( 'Round', 'elementor' ),
+					'rounded' => esc_html__( 'Rounded', 'elementor' ),
+					'sharp'   => esc_html__( 'Sharp', 'elementor' ),
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .your-class' => 'border-style: {{VALUE}};',
+				// ],
+			]
+		);
+
+		$this->add_control(
+			'hr',
+			[
+				'type' => Controls_Manager::DIVIDER,
+			]
+		);
+
+		$this->add_control(
+			'cta_links_padding',
+			[
+				'label'      => esc_html__( 'Padding', 'elementor' ),
+				'type'       => Controls_Manager::DIMENSIONS,
+				'size_units' => [ 'px', '%', 'em', 'rem' ],
+				'default'    => [
+					'top'      => 2,
+					'right'    => 0,
+					'bottom'   => 0,
+					'left'     => 0,
+					'unit'     => 'px',
+					'isLinked' => false,
+				],
+				// TODO: add class selector when markup is done
+				// 'selectors' => [
+				// 	'{{WRAPPER}} .your-class' => 'margin: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				// ],
+			]
+		);
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'background_border_section_style',
+			[
+				'label' => esc_html__( 'Background and Border', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'background_border_background',
+			[
+				'label'     => esc_html__( 'Background', 'elementor' ),
+				'type'      => Controls_Manager::HEADING,
+				'separator' => 'before',
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Background::get_type(),
+			[
+				'name'           => 'background_border_background_group',
+				'types'          => [ 'classic', 'gradient' ],
+				'exclude'        => [ 'image' ],
+				// 'selector' => '{{WRAPPER}} .elementor-button',
+				'fields_options' => [
+					'background' => [
+						'default' => 'classic',
+					],
+				],
+			]
+		);
+
+		$this->add_control(
+			'background_border_background_overlay',
+			[
+				'label'     => esc_html__( 'Background Overlay', 'elementor' ),
+				'type'      => Controls_Manager::HEADING,
+				'separator' => 'before',
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Background::get_type(),
+			[
+				'name'           => 'background_border_background_overlay_group',
+				'types'          => [ 'classic', 'gradient' ],
+				'exclude'        => [ 'image' ],
+				// 'selector' => '{{WRAPPER}} .elementor-button',
+				'fields_options' => [
+					'background' => [
+						'default' => 'classic',
+					],
+				],
+			]
+		);
+
+		$this->end_controls_section();
+
+	}
+
+	private function add_advanced_tab() {
+		Controls_Manager::add_tab(
+			'advanced-tab-custom',
```

Comment:
```suggestion
			static::ADVANCED_TAB,
```
Please use a `const` for this

Reply:
done